shell/composables/useFeatureFlag.ts

```diff
@@ -0,0 +1,21 @@
+import { computed } from 'vue';
+import { Store } from 'vuex';
+import semver from 'semver';
+
+import { getVersionInfo } from '@shell/utils/version';
+
+let store: Store<any>;
+
+export const useFeatureFlag = (vuexStore: Store<any>) => {
+  store = vuexStore;
+
+  return { featureDropdownMenu };
+};
+
+const featureDropdownMenu = computed(() => {
```

Comment:
this should have a descriptive jsdoc comment, or if VS code isn't smart enough to work out the intellisense where it's used then on the json object returned in useFeatureFlag